Src/Plugins/ValidatorBase.cs

```diff
@@ -144,6 +146,36 @@ public static string ReturnUnknownHost(ref string message, string host)
             return nameof(ValidationState.Unknown);
         }
 
+        public static string ParseExpression(IRegex regexEngine, string matchedPattern, string expression)
+        {
+            string pattern = regexEngine.Match(matchedPattern, expression).Value;
+            return ParseValue(pattern);
+        }
+
+        internal static string ParseValue(string value)
+        {
+            if (string.IsNullOrEmpty(value))
+            {
+                return value;
+            }
+
+            value = value.Substring(value.IndexOf('=') + 1);
+
+            int index = 0;
```

Comment:
This code seems intended find the index of the first non-whitespace character, so we can use it to make a substring below.  Why not just trim?  Trim hits both sides if I'm not mistaken.  And we use it anyway.

Reply:
removed that part and added trim only

---
In reply to: [569751725](https://github.com/microsoft/sarif-pattern-matcher/pull/160#discussion_r569751725) [](ancestors = 569751725)